packages/imitation_game/README.md

```diff
@@ -0,0 +1,89 @@
+# Imitation Game
+
+## Description
+
+`imitation_game` is a platform for performing automated tests to compare the
+performance of different UI frameworks.  For example, how much memory does the
+same app written in Flutter and UIKit take?
+
+## Running all the tests
+
+You need a mobile device plugged into your computer and setup for development.
+The mobile device and the computer need to be on the same network, one that
+allows communication between computers since that's how the mobile phone will
+report its results to the computer.
+
+```sh
+dart imitation_game.dart
+```
+
+## Dependencies
+
+In order to run the tests you will need the union of all the platforms being
+tested.  As new tests are added please add to this list:
+
+### iOS
+
+- Flutter
+- Xcode
+- [ios_deploy](https://github.com/ios-control/ios-deploy) - used to launch apps
+  on the attached iOS device.
+
```

Comment:
I'd list out the currently existing scenarios tested and a brief description of the scenarios

Reply:
That information should exist in the readme file for each test.  I don't want it duplicated here.